docs/src/define-routes/https.md

```diff
@@ -186,94 +180,4 @@ tls:
 
 ## Troubleshooting
 
-For easier troubleshooting, feel free to use [the certificate checker tool](https://certcheck.pltfrm.sh/).
```

Comment:
:bulb: It seemed more meaningful to move that part to the general troubleshooting for it to be easier to find and update.

Reply:
Not necessarily opposed to this, but imho I believe it would be better to remove the **Troubleshooting** section then (since it's empty apart from that one link). Instead, I'd suggest adding a link to the general **Troubleshooting** section somewhere in the **TLS configuration** section.

Reply:
Okey yeah that makes sense, let me do that!